### PR TITLE
Audit fixes

### PR DESCRIPTION
**Pull request by prateekreddy**
Audit findings from Peckshield

### CODE REVIEW

File: contracts/Repayments/Repayments.sol [hunk 1]
```diff
@@ -407,7 +407,7 @@ contract Repayments is Initializable, RepaymentStorage, IRepayment, ReentrancyGu
     }
     */
     function getTotalRepaidAmount(address poolID) external view override returns (uint256) {
-        return repaymentVars[poolID].totalRepaidAmount;
+        return repaymentVars[poolID].loanDurationCovered.mul(getInterestPerSecond(_poolID)).div(10**60);
```
Comment (ritik99) [inline contracts/Repayments/Repayments.sol]: could margin calls cause an issue here? For eg, borrower repays some amount, but then after that the lender liquidates via margin call, so then the getInterestPerSecond() function calculates lesser amount than was actually repaid by the borrower

File: contracts/Repayments/Repayments.sol [hunk 2]
```diff
@@ -407,7 +407,7 @@ contract Repayments is Initializable, RepaymentStorage, IRepayment, ReentrancyGu
     }
     */
     function getTotalRepaidAmount(address poolID) external view override returns (uint256) {
-        return repaymentVars[poolID].loanDurationCovered.mul(getInterestPerSecond(_poolID)).div(10**60);
+        return repaymentVars[poolID].loanDurationCovered.mul(getInterestPerSecond(poolID)).div(10**60);
```
Comment (ritik99) [inline contracts/Repayments/Repayments.sol:364]: args will have an underscore acc. to the convention we're following na

**Review by ritik99 — COMMENTED** [review]
two other minor comments: savingsAccount and penalty & savingAccount and penality in some places. Let's just stick to the first convention I guess

File: contracts/Repayments/Repayments.sol (hunk 1 above)
Comment (prateekreddy) [reply contracts/Repayments/Repayments.sol]: Right, makes sense. Do you think we should maintain a different variable for this ?

Comment (prateekreddy) [reply contracts/Repayments/Repayments.sol]: Added a repaidAmount variable to track repayments to a pool

**Pull request merged**